packages/scene-composer/src/components/panels/SettingsPanel.tsx

```diff
@@ -113,12 +136,13 @@ export const SettingsPanel: React.FC<SettingsPanelProps> = ({ valueDataBindingPr
         </ExpandableInfoSection>
       )}
 
-      {tagResizeEnabled && (
+      {(tagResizeEnabled || overlayEnabled) && (
         <ExpandableInfoSection
           title={intl.formatMessage({ description: 'ExpandableInfoSection Title', defaultMessage: 'Tag Settings' })}
           defaultExpanded={false}
         >
-          <SceneTagSettingsEditor />
+          {tagResizeEnabled && <SceneTagSettingsEditor />}
```

Comment:
Component name is bit confusing to me as we have a separate section on the setting which says **Scene Settings** . We have another section called Tag Settings. Can we call this component name as Tag Settings?

Reply:
I'd rather think the Scene Settings is a bit too broad because essentially every settings in the settings panel is a scene setting that impacts the whole scene. and SceneTagSettings describes that this setting is for all tags in the scene, not just one tag, which I think is ok